queries/python/highlights.scm

```diff
@@ -133,7 +133,7 @@
 (none) @constant.builtin
 [(true) (false)] @boolean
 ((identifier) @variable.builtin
- (#lua-match? @variable.builtin "^self$"))
+ (#eq? @variable.builtin "self"))
```

Comment:
Looks like this query is already highlighting self everywhere, so we don't need the other queries that make reference to self

![Screenshot from 2021-12-18 14-23-45](https://user-images.githubusercontent.com/4975310/146653379-41bb8f47-2930-4a76-b0ec-9fc07bff2f5c.png)

as you can see there, the capture is duplicated.

Reply:
True, though this `self` highlight is a bit to broad, as it highlight every instance of `self`, even in free standing functions or the global scope. 

But there is no way around it because of limitations of the query system, AFAIK. So you'd suggest removing these lines?

https://github.com/nvim-treesitter/nvim-treesitter/blob/7cd2ce461b4d2e055d6d7a4063476a540f51422d/queries/python/highlights.scm#L287-L297

Reply:
yeah, since they don't really change anything.

Reply:
Done.